src/zc/lockfile/__init__.py

```diff
@@ -102,3 +103,61 @@ def close(self):
             _unlock_file(self._fp)
             self._fp.close()
             self._fp = None
+
+
+class LockedContext(object):
+    """ContextManager version of LockFile"""
```

Comment:
I think we're missing entries in `CHANGES.rst` for these, and it might be nice if the docstrings included `.. versionadded:: 1.6`.